Source Code/Ticket Management System/Input_manip.c

```diff
@@ -0,0 +1,22 @@
+#include  "Input_manip.h"
+
+wchar_t* input_wchar(wchar_t** input ,int max_characters)
+{
+	wchar_t* context;
+	fgetws(input, max_characters, stdin);
+	fflush(stdin); //clears the buffer just in case there is input that exceeds the limit.
+	wcstok(input, L"\n",&context);	
+	return input;
```

Comment:
Indirection issue.
Did you mean `return *input`?

Reply:
yeah I think this is a mistake, anyway the return can be removed I realized after committing that the functions can be void since I don't really use the return value, but change the pointer.